index.html

```diff
@@ -71,5 +71,99 @@ <h4 id="job">Software Developer</h4>
       <a href=""><p class="work">&#8595; Work</p></a>
     </div>
     </section>
+    <!-- Works Section-->
+    <section id="Projects">
+      <div id="work-intro">
+        <h3 id="work-title">MY RECENT WORK</h3>
+      </div>
+    <section id="work-grid-one">
+      <div class="project-grid">
+        <img class="project-img" alt="project pic" src="https://schemazone.com/wp-content/uploads/2021/03/Project-Manager.jpg">
+          <div class="img-text">
+            <h3 class="project-title">Multi-post Stories</h3>
+             <p class="project-intro">A daily selection of privately personalized reads;
+             no accounts or sign-ups required.
+             has been the industry's standard dummy text ever since the 1500s,
+             when an unknown printer took a standard dummy text.</p>
+           <ul class="tag-list">
+             <li class="tag">css</li>
+             <li><span class="divider"></span></li>
+             <li class="tag">html</li>
+             <li><span class="divider"></span></li>
+             <li class="tag">Bootsrap</li>
+             <li><span class="divider"></span></li>
+             <li class="tag">Ruby</li>
+          </ul>
+          <button type="button" class="btn" onclick="">See project</button>
```

Comment:
Kindly place your cards in the center you can achieve this by adding a padding-left. Please follow and inspect the figma template you picked.
This is your design:
![Screenshot from 2022-06-30 19-46-52](https://user-images.githubusercontent.com/79506105/176723428-1a006537-339b-415c-ad93-82dac2e94af0.png)

Please stick to this design: 
![Screenshot from 2022-06-30 19-47-09](https://user-images.githubusercontent.com/79506105/176723728-8de7af1c-485c-4247-8787-5c7b487724ce.png)

- Please increase the size of the pictures in your  cards follow the figma design

Reply:
The size of the image is exactly as it is in figma, if  I make it any wider, it won't fit on the minimum mobile screen width. 
I have centered the cards.
I put my own image in place of the blank white image in the design.
I will not be adding the screen shot of the UI , I will try make a gif of the website in action later on.